deploy/helm/sumologic/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  repository: sumologic/kubernetes-fluentd
+  tag: 0.0.0
+  pullPolicy: Always
+
+nameOverride: ""
+fullnameOverride: ""
+
+deployment:
+  replicaCount: 3
+  resources: 
+    # We usually recommend not to specify default resources and to leave this as a conscious
```

Comment:
should we remove these comments? the "We" here is confusing since this comment isn't coming from Sumo Logic.

Reply:
removed